### PR TITLE
Santosh nov 04 2019.dev subscription microsoft.subscription 2019 10 01 preview

### PR DESCRIPTION
**Pull request by santoshsinha100**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by santoshsinha100** [comment]
DoNotMerge the pull request

File: specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json [hunk 1]
```diff
@@ -0,0 +1,388 @@
...
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Subscription/SupportPlan": {
+      "get": {
+        "operationId": "GetSupportPlan",
```
Comment (navysingla) [inline specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json]: operationID should be in the form SupportPlan_Get

File: specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.Subscription/PurchaseSupportStatus/{Id}": {
+      "post": {
+        "operationId": "Purchase_SupportStatus",
+        "x-ms-examples": {
+          "enableSubscription": {
```
Comment (navysingla) [inline specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json]: enablesubscription?

File: specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json [hunk 3]
```diff
@@ -0,0 +1,388 @@
...
+          "description": "Error code",
+          "type": "string"
+        },
+        "message": {
+          "description": "Error message indicating why the operation failed.",
+          "type": "string"
+        }
+      }
+    },
+    "CanceledSubscriptionId": {
```
Comment (navysingla) [inline specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json]: Not using this parameter

File: specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json [hunk 4]
```diff
@@ -0,0 +1,388 @@
...
+      "description": "The ID of the canceled subscription",
+      "properties": {
+        "value": {
+          "description": "The ID of the canceled subscription",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "RenamedSubscriptionId": {
```
Comment (navysingla) [inline specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json]: Not using this one

File: specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json [hunk 5]
```diff
@@ -0,0 +1,388 @@
...
+          "type": "string",
+          "description": "the detailed error code"
+        },
+        "billingTotal": {
+          "type": "string",
+          "description": "the billing total"
+        }
+      }
+    },
+    "OperationListResult": {
```
Comment (navysingla) [inline specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json]: Not using this one

File: specification/subscription/resource-manager/readme.md [hunk 6]
```diff
@@ -4,34 +4,46 @@
...
 
-
 ### Basic Information
+
 These are the global settings for the Subscription API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2019-03-preview
+tag: package-preview-2019-10
```
Comment (navysingla) [inline specification/subscription/resource-manager/readme.md]: Don't change this now, we don't want to create SDK for now

**Pull request closed without merging**